TDD_Training/TDD_Training/FindMultiples.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace TDD_Training
+{
+    public class FindMultiples
+    {
+        public string FindMultiple(int givenNumber)
+        {
+            try
+            {
+                if (givenNumber % 3 == 0 && givenNumber % 5 == 0)
+                {
+                    Console.WriteLine("FizzBuzz");
+                    return "FizzBuzz";
+                }
+                if (givenNumber % 3 == 0)
+                {
+                    Console.WriteLine("Fizz");
+                    return "Fizz";
+                }
+                if (givenNumber % 5 == 0)
+                {
+                    Console.WriteLine("Buzz");
+                    return "Buzz";
+                }
+                Console.WriteLine(givenNumber);
+                return Convert.ToString(givenNumber);
+            }
+            catch (Exception ex)
```

Comment:
You can remove the try-catch here I believe, if your logic won't throw any exception